lexicons/app/bsky/feed/defs.json

```diff
@@ -205,6 +209,15 @@
       "type": "object",
       "properties": {}
     },
+    "skeletonReasonSponsored": {
+      "type": "object",
+      "properties": {
+        "info": {
+          "type": "string",
+          "description": "Extended information about sponsored content, provided by the feed generator for transparency purposes."
```

Comment:
Can probably hold off on changes until the `social-app` PR gets a reply, but quick notes:

- a description on the overall object clarifying that it existing at all means the feed item is sponsored content (we don't have good descriptions like this for `skeletonReasonRepost` or `skeletonReasonPin`, but we should)
- we are slowly but surely trying to lock down every `string` field with a `maxLength` to prevent shenanigans. maybe 2000 bytes (like `feedContext`)?

I'd be interested in considering if this should eventually be an AT-URI field pointing at a "context" record of some type. We are thinking about a generic "annotation" / "more context" mechanism for things like labels. That is still speculative though, and this is an optional field so there is a transition path.

Reply:
Thanks for the feedback! I've added a description and a maxLength as requested in the meantime while waiting for a response on bluesky-social/social-app#7518.